libs/libc/string/Make.defs

```diff
@@ -43,6 +47,26 @@ CSRCS += lib_memcpy.c
 endif
 endif
 
```

Comment:
How about adding all the remaining functions that already have Kconfig options created?
- LIBC_ARCH_MEMCMP
- LIBC_ARCH_STRCHR
- LIBC_ARCH_STRLCPY
- LIBC_ARCH_STRNCPY
- LIBC_ARCH_STRNLEN

Reply:
Done.